pkg/tsdb/influxdb/flux/executor_test.go

```diff
@@ -270,3 +269,34 @@ func TestGoldenFiles(t *testing.T) {
 		}
 	})
 }
+
+func TestRealQuery(t *testing.T) {
+	t.Skip() // this is used for local testing
```

Comment:
@aknuds1 -- thoughts on how/if we should include this?

I added this so I could easily start the debugger and step though execution process locally.

Reply:
I will have to check,when time allows.

Reply:
I guess it's OK to keep it like this :shrug: